generic/criu.py

```diff
@@ -52,10 +52,16 @@ def setUp(self):
 
     def test(self):
         os.chdir(self.sourcedir)
-        process.run("./zdtm.py run -a --report sergeyb --keep-going",
+        process.run("./zdtm.py run -a --report sergeyb",
                     ignore_status=True, sudo=True)
         logfile = os.path.join(self.logdir, "stdout")
+        passed_tests = process.system_output(
+            "grep -w PASS %s | wc -l" % logfile, shell=True,
+            ignore_status=True).decode("utf-8")
```

Comment:
Instead of using shell commands for processing. It's better to process the output from the test run using Python.

Reply:
Sure. Can you check below code is that what you are asking or recommended to change:

Reply:
Done